src/communication/http/server.hpp

```diff
@@ -55,12 +48,30 @@ class Server final {
     }
   }
   bool IsRunning() const { return background_thread_ && !ioc_.stopped(); }
-  tcp::endpoint GetEndpoint() const { return listener_->GetEndpoint(); }
+  tcp::endpoint GetEndpoint() const;
```

Comment:
To prevent inlining, which caused template instantiation of `Listener<TRequestHandler, TSessionContext>`, which prevented any benefit of the explicit template instantiation in another translation unit.

Reply:
https://github.com/aras-p/ClangBuildAnalyzer